app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Parent exception for all vaccine-related issues."""
+    pass

```

Comment:
You should add a descriptive error message in the `VaccineError` class. It will be used as a default message for all exceptions that inherit from this class.